modules/weko-index-tree/weko_index_tree/views.py

```diff
@@ -90,3 +91,29 @@ def get_rss_data():
                          count=count,
                          term=term,
                          lang=lang)
+
+
+@blueprint_api.route('/indextree/set_expand', methods=['POST'])
+def set_expand():
+    """Set expand list index tree id."""
+    if current_user.get_id():
+        data = request.get_json(force=True)
+        index_id = data.get("index_id")
+        key = "{}{}".format(
+            current_app.config.get(
+                "WEKO_INDEX_TREE_STATE_PREFIX",
+                WEKO_INDEX_TREE_STATE_PREFIX
+            ),
+            current_user.get_id()
+        )
+        session_data = session.get(key) or []
```

Comment:
I think it is more pythonic to supply the default value in the get() method. 
`session_data = session.get(key, [])`.
Though both are technically the same output.